src/Pages/Login.tsx

```diff
@@ -0,0 +1,112 @@
+import axios from "axios";
+import React, { useState } from "react";
+import { MdEmail } from "react-icons/md";
+import { RiLockPasswordFill } from "react-icons/ri";
+import { Link, useNavigate } from "react-router-dom";
+import { Toast } from "../Components";
+import URL from "../Components/ServerURL";
+
+import { useStore, useUser } from "../Store";
+
+export const Login = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState("");
+
+  const navigate = useNavigate();
+
+  const { userDispatch } = useUser();
+
+  const { storeState, storeDispatch } = useStore();
+  const { loadingMessage } = storeState;
+
+  const loginWithCredentials = async () => {
+    storeDispatch({ type: "IS_LOADING", payload: "loggingIn" });
+    try {
+      const response = await axios.post(`${URL}/login`, {
```

Comment:
here too we can destructure in same manner as before

Reply:
Done